backend/docs/development_docs/api_requests_per_screen.rst

```diff
@@ -0,0 +1,272 @@
+Mapping of backend requests (SOLR queries) to application screens
+=================================================================
+
+This document describes how to get the data necessary for displaying the
+various pages in the phone book mockup. The document will refer to the
+mockups for the web version. These mockups may be found in the same
+directory as the present document, in the file
+``OS2-telefonbog_webversion.pdf``.
+
+The document follows the mockups, describing the necessary queries page
+by page. 
+
+In these examples, the backend (an SOLR server) is assumed to be running
+on localhost (``127.0.0.1``) and port 8983. In production, this will be
+different, and the URLs should be adjusted accordingly.
+
+**Note:** For more advanced search combinations than described in this
+document, there's this quick tutorial:
```

Comment:
I was once told by an English native speaker that they consider the passive voice “weak”…

```suggestion
document, see this quick tutorial:
```